app/controllers/v0/addresses_controller.rb

```diff
@@ -8,8 +8,9 @@ def show
     end
 
     def update
-      # TODO(AJD): raise on missing params, awaiting evss response on req fields
-      response = service.update_address(@current_user, Oj.load(request.body.string))
+      address = EVSS::PCIUAddress::Address.build_address(Oj.load(request.body.string))
```

Comment:
For the models under `/lib/evss/pciu_address` it's to keep everything related to the service, and which shares the same namespace, in one location. Don't have a hard and fast rule but basically anything that could eventually be split out into a micro-service or gem goes under `lib`. EMIS has a similar structure - actually might be better with dirs for class type rather than broken up by sub-service.